mesa/space.py

```diff
@@ -477,9 +482,7 @@ def is_cell_empty(self, pos: Coordinate) -> bool:
         x, y = pos
         return self.grid[x][y] == self.default_val()
 
-    def move_to_empty(
-        self, agent: Agent, cutoff: float = 0.998, num_agents: int | None = None
-    ) -> None:
+    def move_to_empty(self, agent: Agent, num_agents: int | None = None) -> None:
```

Comment:
If you remove the `cutoff` argument, how can you use the new code for benchmark?

Reply:
ok, thinking about it the best way without putting extra non user arguments is to tweak a bit more the code to execute to refer to `self.cutoff_empties` and change that

Reply:
done!